sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluators/_protected_materials/_protected_materials.py

```diff
@@ -89,7 +89,7 @@ class ProtectedMaterialsEvaluator:
         }
     """
 
-    def __init__(self, credential, azure_ai_project: dict):
+    def __init__(self, credential, azure_ai_project):
```

Comment:
ProtectedMaterialsEvaluator not experimental?

Reply:
this evaluator is not exposed, so I will remove it. We have `ProtectedMaterial` evaluator which is public and is experimental